_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -25,32 +25,34 @@ The rule applies to any [keyboard event][]:
 
 ## Expectation 1
 
-For each test target:
- - an [instrument][] to prevent any result of the [keyboard event][] is available; or
- - if the [event target][] doesn't have [focus][], any result of the event is prevented; or
- - an [instrument][] is available to prevent any result of the [keyboard event][] when the [keyboard event][] method `getModifierState` returns `false`.
+For each test target at least one of the following is true:
+ - (**Disable**:) a set of one or more [instruments][instrument] are available to prevent any result of the [keyboard event][] within a 10 minute time span of the [event firing][]; or
```

Comment:
"prevent any result" might be a bit too generic. See discussion (and possible solution) at https://github.com/act-rules/act-rules.github.io/pull/1276#discussion_r436615334 and https://github.com/act-rules/act-rules.github.io/pull/1276/files/621237308c19c3314dc3a7b104db4a9c8cf9678c..2e8eb9a051180f5fc3bc094c2032372e5558f8a0

I thing it might also help tightening the rule to specifically target keyboard shortcuts that create "changes in content"

Being more precise about the changes we monitor will also help killing off the second note as requested by @daniel-montalvo.